requirements/dev.txt

```diff
@@ -14,5 +13,5 @@ hijridate
 # Deployment.
 build
 gitpython
-pip>=19.2.3
+pip
```

Comment:
```suggestion
```
Starting with Python 3.4, it is included by default with the Python binary installers.